inc/files.php

```diff
@@ -522,6 +524,8 @@ function ppom_uploaded_file_preview( $file_name, $settings ) {
 // Trim long filename to short
 function ppom_files_trim_name( $file_name ) {
 
+	return $file_name;
```

Comment:
Was this shortcircuit for debugging only?

Reply:
@Soare-Robert-Daniel  No, I think I missed removing the previous code, now we are displaying the full name.

I will remove it and push the changes

Reply:
@Soare-Robert-Daniel I've removed the code, please check the latest commit.